oat-sa/tao-core
# feature/TR-2758/set-media-interaction-ltr-direction

**Related to:** [TR-2758](https://oat-sa.atlassian.net/browse/TR-2758)

**Description:**

Media Interaction RTL needs to be reverted in Terre, should keep LTR always.

**Changes:**

- Link to the package and bundle files

**How to check:**

- Create a media player interaction test with two items, one in LRT language and the other in RTL language. There is a test attached to the ticket if you want to use it.
- Check that the media interaction changes its alignment depending on the item direction, but the media player buttons keep on LTR mode in every case.
![media_player_rtl](https://user-images.githubusercontent.com/11692751/143581637-35cc3134-bd61-4b4f-857f-2efd4908e025.png)

**TAE Environment:**

- Link to the TAE on the ticket

**Local test:**

- There is a composer link on the ticket to install it locally
